src/sets.jl

```diff
@@ -1534,6 +1534,24 @@ Base.copy(set::Table) = Table(copy(set.table))
 
 Base.:(==)(x::Table{T}, y::Table{T}) where {T} = x.table == y.table
 
+"""
+    Reified(set::AbstractSet)
```

Comment:
With the exception of `GreaterThan`, `LessThan`, and `EqualTo`, all set names are nouns and not modifiers (like adjectives).

Reply:
There's also `Indicator`. Does that count?